x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.ts

```diff
@@ -22,7 +23,15 @@ export function getOpsStatsCollector(
   metrics$: Observable<OpsMetrics>
 ) {
   let lastMetrics: MonitoringOpsMetrics | null = null;
-  metrics$.subscribe(metrics => {
+  metrics$.subscribe(_metrics => {
+    const metrics: any = cloneDeep(_metrics);
```

Comment:
nit: I don't think we need to use `cloneDeep` here (unless `metrics` is somehow a reference), and looks like `_metrics` is not used anywhere else. Don't know if Observables hold references though, in which case maybe shallow might be good enough?:
`metrics$.subscribe(({ ...metrics }) => {`

Only mentioning because I know `cloneDeep` can be pretty expensive

Reply:
Yea I tried it without it and any `delete` or re-setting the object caused issues with the other subscriber.